source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+- Are there any known security or access requirements?
+- Is a VPN connection required?
+- Do we use an Enterprise Mobile Management (EMM) Provider?
+
+Again, this means pausing before you jump into your Mattermost mobile deployment. This takes a little more time, but will make the difference in ensuring your see a return on your investment.
+
+----
+
+###################################
+Mattermost Mobile Deployment Primer
+###################################
+
+A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
+
+We’re here to support you.
+
+This primer serves as an introduction to Mattermost Mobile technologies. It is a compliment to our technical documentation, and focuses on three main areas:
+
+- Technical Requirements
+- Basic Features and Technologies
+- Trouble Spots
+
+Regardless of your expertise, we recommend reading this section before starting your deployment.
+
+***********************
+Technical  Requirements
+***********************
+
+`Supported Mattermost Server Versions <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`_ 
+    Minimum requirements for the Mattermost Server is maintained in our documentation (link above). 
+    **However, we do recommend running the latest version** of the Mattermost Server as it contains the most recent features and any applicable security updates.
+
+    If this is not possible, we encourage you to bon at least version 5.12 of the Mattermost Server. This release has a number of critical feature updates that will ensure compatibility in a number of areas, including the Mattermost push notification service (MPNS).
+
+`Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`_
+    Basic mobile device requirements are provided in the link above.
+
+`Network Requirements <https://>`_
+
+********************************
+Public App Store vs. Custom Apps
+********************************
+
+The most critical decision you will make in your mobile deployment will be whether to use the apps provided by Mattermost via the public App Stores (i.e. Google Play And Apple App Store), or to build and distribute custom versions of the Mattermost apps. Below we provide a very general overview of these options.
+
+Public App Store
+================
+
+Utilizing the public app stores greatly reduces the deployment time and is our recommended approach. Key benefits include… 
+
+- Easier deployment, driven by user needs.
+- Ability to use the Mattermost Push Notification Service, or MPNS.
+- Apps automatically updated with the latest features, enhancements and security updates.
+
+Again, this is the recommended route which is as simple as downloading the Mattermost mobile application from the corresponding store, entering a URL to point towards your Mattermost server, and logging in.
+
+Custom Apps
```

Comment:
```suggestion
Custom Builds
```

Reply:
-1 This follows the language in the docs: “Follow these steps to receive push notifications in your custom app.“ https://developers.mattermost.com/contribute/mobile/build-your-own/

Reply:
I just don't feel the time involved in making these changes really adds a bunch of value. There's no evidence anyone is confused, and at the very least this is the internal and existing language.

Reply:
solved in meeting